kustomize/gcp/database/cloudsqlinstanceclass.yaml

```diff
@@ -35,7 +35,7 @@ specTemplate:
     databaseVersion: POSTGRES_9_6
     region: $(REGION)
     settings:
-      tier: db-n1-standard-1
+      tier: db-custom-1-3840
```

Comment:
this looks like the right "custom" machine type for the given predefined machine type 👍